styles.css

```diff
@@ -1,8 +1,71 @@
-/* font-family: 'Quicksand', sans-serif;
+ /* font-family: 'Quicksand', sans-serif;
```

Comment:
CSS looks good. If you ever see yourself using the same thing frequently like you are with `font-family: 'Quicksand', sans-serif;`

Consider the CSS design pattern mentioned above where you can have a single css rule for it and just add that class to it so have.

.quicksand {
    font-family: 'Quicksand', sans-serif;
}

.radio-box {
  border-radius: 10px;
  margin: 100px auto 10px auto;
  height:90px;
  width:575px;
  background-color: #ffffff
}

then have < input type="radio" class="radio-box quicksand" >

Reply:
This is very helpful, I'll see where I can reduce the need for repetitive styling.